src/api/app/views/shared/_package_branch_form.html.haml

```diff
@@ -1,23 +1,26 @@
+- if show_project_field
+  %p
+    %strong Change Branch Destination
+  %p
+    %label{ for: 'target_project' }
+      %strong Branch project name
+    = text_field_tag 'target_project', target_project, size: 80, maxlength: 200
 %p
-  - if show_source_fields
-    %label{ for: 'linked_project' }
-      %strong Name of original project:
-    %br/
-    = text_field_tag 'linked_project', nil, size: 80, id: 'linked_project', required: true
-    %br/
-    %label{ for: 'linked_package' }
-      %strong Name of package in original project:
-    %br/
-    = text_field_tag 'linked_package', nil, size: 80, id: 'linked_package', required: true
-    %br/
-  - else
-    = hidden_field_tag(:linked_project, @project.name)
-    = hidden_field_tag(:linked_package, @package.name)
   %label{ for: 'target_package' }
-    %strong New package name:
+    %strong Branch package name
   = text_field_tag 'target_package', @package.try(:name), size: 80, maxlength: 200
-  %br/
-  = hidden_field_tag 'target_project', target_project.name if target_project
-  %br/
+%p
   = check_box_tag 'current_revision', false
   %label{ for: 'current_revision' } Stay on current revision
+  - if @revision
+    (##{@revision})
+- if show_project_field
```

Comment:
This could be something like:
``` ruby
 if show_project_field && Configuration.cleanup_after_days
```

Reply:
the part after the `&&` doesn't get evaluated if the first part is `false`, right? In that case you are right, I should join them together.